quanto/tensor/optimizers/mse_optimizer.py

```diff
@@ -0,0 +1,72 @@
+from typing import Tuple, Union
+
+import torch
+from torch import Tensor
+
+from quanto.tensor.core import axis_to_dim
+
+from .affine_optimizer import AffineOptimizer
+from .symmetric_optimizer import SymmetricOptimizer
+
+
+def _fake_quantize(base: Tensor, scale: Tensor, zeropoint: Union[int, Tensor], qmin: int, qmax: int) -> Tensor:
+    data = base / scale + zeropoint
+    data = torch.round(data)
+    data = torch.clamp(data, qmin, qmax)
+    data = (data - zeropoint) * scale
+    return data
+
+
+class MseSymmetricOptimizer(SymmetricOptimizer):
+    def __init__(self, p: float = 2.0, iters: int = 80) -> None:
+        self.iters = iters
+        self.p = p
+        assert p >= 1.0
+        assert 0 < iters < 100
+
+    def optimize(self, base: Tensor, bits: int, axis: int) -> Tensor:
```

Comment:
You are using the AbsMax as a starting point, then iterate: please do it explicitly to avoid code duplication.
You can for instance inherit from AbsmaxOptimizer and call super().optimize, or simply instantiate an AbsmaxOptimizer in your code.